test/index.js

```diff
@@ -24,4 +24,4 @@ import './lib';
 import './utils';
 import './experimental';
 import './deprecated';
-import './debug/seer-integration';
+import './debug/seer-integration.spec';
```

Comment:
> why changing this? we are already in the tests directory

Every other file in the test directory follows this `.spec` convention.

This convention is pretty common in the JS community. FWIW, it does have the advantage of easily distinguishing the two files e.g. in sublime tabs or when do command-open in the Chrome debugger.